BIP ??: Opt-in Anti-Replay via Coinbase Generated Outputs (OP_ANTI_REPLAY)

da2ce7 opened the pull request:
Design of soft-fork that provides opt-in consensus-based protection from anti-replay attacks.
Hard-forking alt-chains would have to be explicitly malicious to avoid these anti-replay protections.

luke-jr commented:
I'm not convinced this meets the technical soundness requirement. [Let's discuss on the ML?](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-March/013845.html)

da2ce7 commented:
Thank you for your feedback on the mailing list. I will close the pull request until I have addressed all the comments.

Many Thanks.


Sent from my iPhone

> On 29 Mar 2017, at 12:17, Luke Dashjr <notifications@github.com> wrote:
> 
> I'm not convinced this meets the technical soundness requirement. Let's discuss on the ML?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>
